data/reusables/issue-events/assignee-properties.md

```diff
@@ -1,2 +1 @@
 `assignee` | `object` | The person unassigned from this issue.
```

Comment:
Thanks for realizing that this field exists for the Issue events API but not the Timeline events API! With that in mind, maybe we should keep this line but add that it is only available for one of the APIs. How does this sound?

```suggestion
`assignee` | `object` | The person unassigned from this issue.
`assigner` | `object` | The person who performed the unassignment for this issue. This field is available in the Issue events API but not the Timeline events API.
```